variables.tf

```diff
@@ -0,0 +1,139 @@
+variable "location" {
+  description = "Location to deploy to"
+  type        = string
+}
+
```

Comment:
unless there is a specific reason we prefer to use locals over variables in the main module